node/src/board.js

```diff
@@ -64,8 +64,21 @@ let boardModule = {
                     }
                 }
             }
+            // Si la carte ne peut pas être placée (Exemple sa carte est 1 donc aucune place pour elle), on supprime 
```

Comment:
Comments are usually code smells. They should explain why you do what you do, not what you actually do. Code should be self explanatory.
Here, I don't understand anything of the comment or what the code below does.